GameServerLib/Config.cs

```diff
@@ -69,12 +69,15 @@ private void LoadConfig(Game game, string json)
             // Read where the content is
             ContentPath = (string)gameInfo.SelectToken("CONTENT_PATH");
 
+            // Load items
+            game.ItemManager.LoadItems(ContentPath);
```

Comment:
What's up with this addition?

Reply:
It was moved from somewhere upper in the execution order to there to prevent issues...
I think.